File: scripts/_check_tabs.py
import json
import sys
args = json.loads(sys.argv[1]) if len(sys.argv) > 1 else {}
main = document.querySelector('main') or document.body
txt = main.innerText or ''
out = txt[-400:] if txt else 'EMPTY'
return json.dumps({'len': len(txt), 'tail': out})

File: scripts/_check_tabs.js
// Check last 400 chars of main content
var main = document.querySelector('main') || document.body;
var txt = main.innerText || '';
var out = txt.length > 400 ? txt.slice(-400) : txt;
return JSON.stringify({len: txt.length, tail: out.replace(/\n/g, ' | ')});